module/platform/darwin/SensorFilter/src/SensorFilter.cpp

```diff
@@ -517,6 +518,8 @@ namespace platform {
                         else if (!foot_down) {
                             previous_foot_down[side] = false;
                         }
+
+                        sensors->feet[side].Hwf = footlanding_Hwf[side];
```

Comment:
So when our foot is not on the ground here I assume that means it'll just contain whatever the last position it had on the ground was

Reply:
Every sensors update will have the most recent Hwf for both feet